go/teams/teams.go

```diff
@@ -850,3 +874,35 @@ func LoadTeamPlusApplicationKeys(ctx context.Context, g *libkb.GlobalContext, id
 	}
 	return team.ExportToTeamPlusApplicationKeys(ctx, keybase1.Time(0), application)
 }
+
+func (t *Team) loadAllTransitiveSubteams(ctx context.Context) ([]*Team, error) {
+	subteams := []*Team{}
+	for subteamID := range t.Data.Chain.SubteamLog {
```

Comment:
This made me think of a weird bug that exists in the loader now. So definitely use `ListSubteams` and you will inherit the fix when it happens.